src/app/sold-products/sold-products.component.spec.ts

```diff
@@ -17,9 +27,29 @@ describe('SoldProductsComponent', () => {
     fixture = TestBed.createComponent(SoldProductsComponent);
     component = fixture.componentInstance;
     fixture.detectChanges();
+    service = TestBed.inject(ProductService);
   });
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+ 
+  it('service should be injected', () => {
+    expect(service).toBeTruthy();
+  });
+
+  it('should get products$ Observable', (done) => {
+    component.products$.subscribe(value => {
+      expect(value).toBeTruthy();
+      done();
+    });
+  });
+
+  it('should get targetValue$ Observable', (done) => {
+    component.targetValue$.subscribe(value => {
```

Comment:
No need to test RxJS

Reply:
So how to cover this part of `ng test --no-watch --code-coverage`: 
![image](https://user-images.githubusercontent.com/23476751/107349000-fe27f600-6ac7-11eb-8084-21c073771a22.png)

Reply:
Coverage is not always the most important factor. If you want to check this check edge cases with some logic if possible. This way you only test RxJS map function which is tested in the framework.

Reply:
Understood, test removed.